code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -397,18 +397,14 @@
 	return
 
 
-///checks for any non-anchored movable atom, throwing them towards the shrike/owner using the ability. While causing shake to anything in range with effects applied to humans affected.
+/**
+ * Checks for any non-anchored movable atom, throwing them towards the shrike/owner using the ability. 
+ * While causing shake to anything in range with effects applied to humans affected.
+ */
 /datum/action/xeno_action/activable/psychic_vortex/proc/vortex_pull()
 	playsound(owner, 'sound/effects/seedling_chargeup.ogg', 60)
-	for(var/atom/victim AS in range(VORTEX_RANGE, owner.loc))
-		if(isturf(victim))
-			continue
-		if(!ismovableatom(victim))
-			continue
-		var/atom/movable/movable_victim = victim
-		if(movable_victim.anchored)
-			continue
-		if(isxeno(movable_victim))
+	for(var/atom/movable/movable_victim in range(VORTEX_RANGE, owner.loc))
+		if(movable_victim.anchored || isxeno(movable_victim) || movable_victim.move_resist == INFINITY)
```

Comment:
```suggestion
		if(movable_victim.anchored || isxeno(movable_victim) || movable_victim.move_resist > MOVE_FORCE_STRONG)
```
Up to you, but it might be worth allowing humans to strap in to avoid getting thrown about.